From iwamatsu@nigauri.org Wed Oct 10 08:38:35 2007
From: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>
Date: Thu, 11 Oct 2007 00:37:50 +0900
Subject: USB: Adding support for SHARP WS011SH to ipaq.c
To: linux-usb-devel@lists.sourceforge.net
Cc: <iwamatsu@nigauri.org>, Ganesh Varadarajan <ganesh@veritas.com>, Greg KH <greg@kroah.com>
Message-ID: <20071011003750.92603e93.iwamatsu@nigauri.org>


This patch supports for SHARP WS011SH[0] to ipaq.c

[0]: http://www.willcom-inc.com/ja/lineup/ws/011sh/index.html
(Sorry , Japanese only.)

Signed-off-by: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/ipaq.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/serial/ipaq.c
+++ b/drivers/usb/serial/ipaq.c
@@ -256,6 +256,7 @@ static struct usb_device_id ipaq_id_tabl
 	{ USB_DEVICE(0x04DD, 0x9121) }, /* SHARP WS004SH USB Modem */
 	{ USB_DEVICE(0x04DD, 0x9123) }, /* SHARP WS007SH USB Modem */
 	{ USB_DEVICE(0x04DD, 0x9151) }, /* SHARP S01SH USB Modem */
+	{ USB_DEVICE(0x04DD, 0x91AC) }, /* SHARP WS011SH USB Modem */
 	{ USB_DEVICE(0x04E8, 0x5F00) }, /* Samsung NEXiO USB Sync */
 	{ USB_DEVICE(0x04E8, 0x5F01) }, /* Samsung NEXiO USB Sync */
 	{ USB_DEVICE(0x04E8, 0x5F02) }, /* Samsung NEXiO USB Sync */
